Vector/ViewController/RecentsViewController.m

```diff
@@ -397,6 +438,31 @@ - (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSIntege
     return [(RecentsDataSource*)self.dataSource heightForHeaderInSection:section];
 }
 
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    if (self.delegate)
```

Comment:
I suggest retrieving the selected cell and ignore selection if its class is InviteRecentTableViewCell.class